module/Makefile.in

```diff
@@ -6,6 +6,7 @@ subdir-m += spl
 subdir-m += unicode
 subdir-m += zcommon
 subdir-m += zfs
+subdir-m += zstd_zfs
```

Comment:
should this go before `zfs`?

Reply:
You're correct, it should.

Reply:
I am willing to bet this is the one that is causing the concurrency issues...

Reply:
@allanjude pretty sure, yes

Reply:
and now you've got that error in the tests as well ;)

Reply:
Even with zstd_zfs before zcommon, same build failure. I cannot recreate this on a completely fresh build, but somehow can get it to happen when rebuilding incrementally later.